code/datums/traits/negative.dm

```diff
@@ -424,6 +424,39 @@
 	to_chat(quirk_holder, "<span class='boldannounce'>Your [slot_string] has been replaced with a surplus prosthetic. It is fragile and will easily come apart under duress. Additionally, \
 	you need to use a welding tool and cables to repair it, instead of bruise packs and ointment.</span>")
 
+/datum/quirk/nugget
+	name = "Nugget"
+	desc = "An accident caused you to lose ALL of your limbs. There's no way your insurance payed for all those prosthetics!"
+	value = -1
+	var/slot_string = "limb"
+
+/datum/quirk/nugget/on_spawn()
+	var/mob/living/carbon/human/nuggeted = quirk_holder
+	for(var/X in nuggeted.bodyparts)
```

Comment:
```suggestion
	for(var/obj/item/bodypart/BP in nuggeted.bodyparts)
```
There's a built-in type check, you don't need that second variable

Reply:
compiler would cry would it not? its calling a variable thats exclusive to a bodypart afterwards without an implicit type, unless byond is smart enough to see the listtype and not error out